libs/api/domains/cms/src/lib/cms.elasticsearch.service.ts

```diff
@@ -175,6 +177,27 @@ export class CmsElasticsearchService {
     return response ? JSON.parse(response) : null
   }
 
+  async getOrganizationSubpage(
+    index: string,
+    { slug, organizationSlug }: GetOrganizationSubpageInput,
+  ): Promise<OrganizationSubpage | null> {
+    // return a single news item by slug
+    const query = {
+      types: ['webOrganizationSubpage'],
+      tags: [
+        { type: 'slug', key: slug },
+        { type: 'organization', key: organizationSlug },
+      ],
+    }
+    const menuResponse = await this.elasticService.getDocumentsByMetaData(
```

Comment:
Is this potentially a copy/paste error? Should this be `menuResponse`?
Should this also maybe use `getSingleDocumentByMetaData`